libs/shared/src/dto/session/session.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { ISubscriberJwt } from '../../entities/user';
+
+export interface ISessionDto {
+  token: string;
+  profile: ISubscriberJwt;
```

Comment:
since this is a subscriber object let’s stick to that terminology

```suggestion
  subscriber: ISubscriberJwt;
```

Reply:
It's the interface for the existing Dto class, we can do refactoring in the separate PR as it will require more changes.

Reply:
Agreed, but let's add a task so as not to forget it as the `profile` attribute is confusing.

Reply:
I already created one: https://linear.app/novu/issue/COM-33/refactor-the-session-interface ;)